README.md

```diff
@@ -175,44 +175,6 @@ Impact the product, hear updates, test drive new features and more! Join [here](
 
 Released under the MIT license.
 
-## Development
+## Contributing
```

Comment:
Can you move this before Additional Resource section and change the title to How to contribute? Thanks @arturkulig

Reply:
Done 👍